### ISSUE TITLE
Changing categories of "Browse"

### ISSUE THREAD

**Issue by sezanzeb** [flaviotordini/minitube#187]
The animation on https://flavio.tordini.org/minitube shows various other categories, while on my installation it's only 6.

They also have this black border around them
![image](https://user-images.githubusercontent.com/28510156/104138215-f4b14e00-53a2-11eb-953e-bfcc604c4a3e.png)

while I would prefer it to be the background color of the window like in
![image](https://user-images.githubusercontent.com/28510156/104138282-5f628980-53a3-11eb-8708-c057a782bf06.png)

And ideally I would like to configure those categories myself or to have a neat overview of random videos of my subscriptions, like a front page.

**Comment by flaviotordini** [comment]
Yes, implementation changed and so it is a bit different from screenshot. Having it configurable would be cool, like saved searches.
